views/components/tabs.njk

```diff
@@ -1,14 +1,14 @@
 {% from "components/icons.njk" import iconArrowDown %}
 
 {% macro tab(title, paneId, tabClasses = false, setActive = false, id = false) %}
-    <a class="js-tab{% if tabClasses %} {{ tabClasses }}{% else %} tab{% endif %}{% if setActive %} tab--active{% endif %}"
+    <a role="tab" class="js-tab{% if tabClasses %} {{ tabClasses }}{% else %} tab{% endif %}{% if setActive %} tab--active{% endif %}"
```

Comment:
This ~might~ break on the data page where tabs are used out of context, but improves things overall so will come-back and fix that if needs be.